charts/cht-user-management/Chart.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   - alias: cht-user-management
     name: base-single-app
     version: "0.2.0"
-    repository: file://../base-single-app
+    repository: https://docs.communityhealthtoolkit.org/helm-charts
```

Comment:
Can this be `communityhealthtoolkit.org/helm-charts` instead of `docs.communityhealthtoolkit.org/helm-charts`?

Reply:
Prior conversation on [slack](https://medic.slack.com/archives/C02F2TAME90/p1704983610436129?thread_ts=1704903060.183429&cid=C02F2TAME90)
I don't think it should use the top level domain of any domain as that would limit what it can be used outside of helm, but I would be all for using another subdomain if possible through gh-pages. Something like `helm.communityhealthtoolkit.org`, `helm-charts.medicmobile.org`?

Reply:
Merging to get the gitignore changes in, but I'm happy to continue the domain conversation, it is best to change it now before it is in use in other places e.g. https://github.com/medic/cht-core/pull/8996